include/model/CAnnotatedProbability.h

```diff
@@ -124,9 +119,6 @@ struct MODEL_EXPORT SAnnotatedProbability {
     SAnnotatedProbability();
     SAnnotatedProbability(double p);
 
-    //! Add the descriptive data \p value for \p key.
-    void addDescriptiveData(annotated_probability::EDescriptiveData key, double value);
```

Comment:
There are some typedefs which should be removed as well. I guess these should surface when you remove `EDescriptiveData`.